internal/pkg/service/buffer/store/model/conditions.go

```diff
@@ -44,7 +44,7 @@ func (c Conditions) Evaluate(now time.Time, openedAt time.Time, s Stats) (bool,
 		return true, fmt.Sprintf("count threshold met, received: %d rows, threshold: %d rows", s.RecordsCount, c.Count)
 	}
 	if s.RecordsSize >= c.Size {
-		return true, fmt.Sprintf("size threshold met, received: %s, threshold: %s", s.RecordsSize.String(), c.Size.String())
+		return true, fmt.Sprintf("size threshold met, received: %s, threshold: %s", s.RecordsSize.HumanReadable(), c.Size.HumanReadable())
```

Comment:
Limit je vacsinou nejaka okruhla hodnota, ale realna velkost je nahodna.
Ak tam pouzijem `String()`, tak mi to vzdy vypise presne = teda v `bytes`, `HumanReadable` to zaokrúhli a bude to v `kB, MB, ...`.